source/stable-release_iot/user-guide-a12.rst

```diff
@@ -0,0 +1,418 @@
+For Android 12 Guest VM only, edit the memory and vcpu sections of the configuration ini file at <workspace>/.intel/.civ/civ-sriov.ini.
+
+.. code-block:: bash
+
+	# Enable USB Devices in Guest VM
+	# [memory]
+	# size=4G
+
+	# [vcpu]
+	# num=4G
+
+.. note::
+	A passthrough device option can only be used once, because a device can be passthrough to only 1 guest VM at a time
+	For Android 12 guest VM, the passthrough is defined in the configuration ini file.
+
+Android 12 guest VM USB device passthrough:
+
+This section describes the steps to run Android 12, ADL platform.
+
+.. code-block:: bash
+
+	# Find the PCI ID of the USB device
+	$ $ lspci -nn -D | grep USB
+		0000:00:14.0 USB controller [0c03]: Intel Corporation Device [8086:7ae0] (rev 11)
+		0000:00:14.1 USB controller [0c03]: Intel Corporation Device [8086:7ae1] (rev 11)
+		0000:05:00.0 USB controller [0c03]: Intel Corporation Thunderbolt 4 NHI [Maple Ridge 4C 2020] [8086:1137]
+		0000:07:00.0 USB controller [0c03]: Intel Corporation Thunderbolt 4 USB Controller [Maple Ridge 4C 2020] [8086:1138]
+
+	# Edit the passthrough section of the configuration ini file at <workspace>/.intel/.civ
+	#[passthrough]
+	#specified the PCI id here if you want to passthrough it to guest, separate them with comma
+	$ passthrough_pci=0000:00:14.0,0000:00:14.1,0000:05:00.0,0000:07:00.0,
+
+
+Enable PCIe Wi-Fi Adapter Device in Guest VM:
+This section describes to enable PCIe
+
+.. code-block:: bash
+
+	$ lspci -nn -D | grep Wi-Fi
+	0000:02:00.0 Network controller [0280]: Intel Corporation Wi-Fi 6 AX210/AX211/AX411 160MHz [8086:2725] (rev 1a)
+	# Then edit the passthrough section of the configuration ini file at <workspace>/.intel/.civ.
+	
+.. note::
+	A passthrough device option can only be used once, because a device can be passthrough to only 1 guest VM at a time
+	For Android 12 guest VM, find the PCI ID of the Wi-Fi device
+
+Enable logging for Android 12 Guest VM:
+This section describes to debug logging
+
+.. code-block:: bash
+
+	# Edit the extra section of the configuration ini file at <workspace>/.intel/.civ.
+	[extra]
+	cmd=-chardev socket,id=ch0,path=/tmp/civ1-console,server=on,wait=off,logfile=/tmp/civ1_serial.log -serial chardev:ch0
+	
+	[passthrough]
+	#specified the PCI id here if you want to passthrough it to guest, separate them with comma
+	passthrough_pci=0000:02:00.0
+
+	# Connect to Android 12 Guest VM console for any debugging
+	# Connect to Celadon guest console
+	$ sudo socat unix-connect:/tmp/civ1-console stdio
+
+Launch Guest VM on Single Display and Full Screen Mode:
+For Android 12 guest VM, edit the extra section of the configuration ini file at <workspace>/.intel/.civ
+
+.. code-block:: bash
+
+	# different according to the use cases.
+	[extra]
+	cmd=-full-screen
+
+.. note::
+	The amount of memory and cores allocated might be different according to each platform. 
+	And the combination of multiple Guest VMs and multiple displays might be 
```

Comment:
I think there is no multiple displays in this particular user guide. Please correct it